File: backend/src/main/java/com/java/NBE4_5_1_7/domain/member/entity/SubscriptionPlan.java
package com.java.NBE4_5_1_7.domain.member.entity;

import lombok.Getter;

@Getter
public enum SubscriptionPlan {
    FREE("FREE",0,0),
    PREMIUM("PREMIUM", 100, 30);

    private final String name;
    private final int price;
    private final int durationDays;

    SubscriptionPlan(String name, int price, int durationDays) {
        this.name = name;
        this.price = price;
        this.durationDays = durationDays;
    }
}

File: backend/src/main/java/com/java/NBE4_5_3_7/domain/member/entity/SubscriptionPlan.kt
package com.java.NBE4_5_3_7.domain.member.entity

enum class SubscriptionPlan(
    val planName: String,
    val price: Int,
    val durationDays: Int
) {
    FREE("FREE", 0, 0),
    PREMIUM("PREMIUM", 100, 30)
}
